test/unit/Reflection/ReflectionFunctionAbstractTest.php

```diff
@@ -349,6 +349,7 @@ public function returnTypeFunctionProvider()
             ['returnsString', 'string'],
             ['returnsNull', 'null'],
             ['returnsObject', '\stdClass'],
+            ['returnsVoid', 'void'],
```

Comment:
Yes, nothing needs to change. phpDocumentor/TypeResolver already supported `void` so nothing needs to change for us, just a new test